fineract-provider/src/main/java/org/apache/fineract/infrastructure/documentmanagement/api/ImagesApiResource.java

```diff
@@ -140,7 +140,8 @@ public Response retrieveImage(@PathParam("entity") final String entityName, @Pat
             imageDataURISuffix = ContentRepositoryUtils.ImageDataURIsuffix.PNG.getValue();
         }
 
-        final String clientImageAsBase64Text = imageDataURISuffix + Base64.encodeBytes(imageData.getContentOfSize(maxWidth, maxHeight));
+        byte[] resizedImage = imageData.getContentOfSize(maxWidth, maxHeight);
+        final String clientImageAsBase64Text = imageDataURISuffix + Base64.getEncoder().encode(resizedImage);
```

Comment:
Base64.getEncoder().encode() returns byte[] 
Are you sure about this string concatenation?

Reply:
@fynmanoj no you're absolutely right of course, it was (very) wrong - Thank You very much for spotting that! 

PS: It's totally OK to help code review and tell me "this is wrong!" more strongly than your very polite "Are you sure about" :smile:

Reply:
@vorburger , Sure 😄 👍